tee-worker/omni-executor/heima/identity-verification/src/web2/google/mod.rs

```diff
@@ -0,0 +1,102 @@
+mod oauth2_client;
```

Comment:
Can this be moved out of identity-verificaiton / heima context ? To me it looks like just general oauth